src/lib/felaExpandCssShorthandsPlugin.ts

```diff
@@ -0,0 +1,47 @@
+import * as _ from 'lodash'
+import * as expand from 'css-shorthand-expand'
+
+export default () => {
+  const expandCssShorthands = styles => {
+    const processedStyles = {}
+
+    Object.keys(styles).forEach(cssPropertyName => {
```

Comment:
nit: if `lodash` is imported and, what is more appealing, is used below, we might consider to use _.keys :) in either case, please, feel free to decide by your own

Reply:
The code is refactored.